recipe/meta.yaml

```diff
@@ -64,6 +65,13 @@ outputs:
         # test pkgconfig metadata
         - pkg-config --print-errors --exact-version "{{ version }}" openssl
 
+        # ensure (de)activation scripts for bash-on-win do not contain crlf line endings;
+        # ripgrep will return exit code 1 if no match is found, which is what we want after
+        # filtering to the carriage-return character that shouldn't be there.
+        {% for task in ["activate", "deactivate"] %}
+        - type %CONDA_PREFIX%\etc\conda\{{ task }}.d\openssl_{{ task }}-win.sh | rg \r & if %ERRORLEVEL% NEQ 1 (exit 0) else (exit 1)  # [win]
```

Comment:
ugh, I managed to combine C&P with a brainfart and produce the wrong ordering here...

Reply:
That this still works is a mystery, but apparently there's some sort of caching or warm up going on:
```
C:\Users>conda activate builder
(builder) C:\Users>type %CONDA_PREFIX%\etc\conda\activate.d\openssl_activate-win.sh | rg \r & echo %ERRORLEVEL%
0
(builder) C:\Users>type %CONDA_PREFIX%\etc\conda\activate.d\openssl_activate-win.sh | rg \r & echo %ERRORLEVEL%
1
(builder) C:\Users>type %CONDA_PREFIX%\etc\conda\activate.d\openssl_activate-win.sh | rg \r & echo %ERRORLEVEL%
1
```
only the first evaluation gives the wrong answer